reference/features/search_parameters.md

```diff
@@ -394,6 +392,8 @@ You would get the following response with **information about the matches in the
 }
 ```
 
+Note how MeiliSearch searches for `winter` and `feast` separately because of the whitespace.
```

Comment:
I think I would keep this together with the content on L#365 so the text flows a bit more naturally.